django/forms/forms.py

```diff
@@ -249,7 +250,8 @@ def _html_output(self, normal_row, error_row, row_ender, help_text_html, errors_
                         'label': '',
                         'field': '',
                         'help_text': '',
-                        'html_class_attr': html_class_attr,
```

Comment:
I saw no real reason for it when creating an empty row for the sole purpose to insert other output. However, I digged through the tickets I could find on this (3512, 3686, 3515), and looked at the [original commit](https://github.com/django/django/commit/92803205cbcaaee16ac0eb724c45019a9d896aac) adding it, and it seems like it was written like that on purpose, although I find no reference to why. So I just pushed a new commit that reverts it, doesn't make a difference for this new patch :)